renovate-config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    extends: ["group:allNonMajor", "config:recommended"],
+    dependencyDashboard: true,
+    repositories: ['zowe/api-layer'],
+    baseBranches: ["v2.x.x","v3.x.x"],
+    printConfig: true,
+    prHourlyLimit: 0, // removes rate limit for PR creation per hour
+    recreateWhen: "always", // recreates all closed or blocking PRs not just immortals
```

Comment:
Didn't you want to change it to default option?

Reply:
Yes, I forgot. Thanks
I mention this option in comment here. https://github.com/zowe/api-layer/pull/3667#issuecomment-2257683987